config/routes.rb

```diff
@@ -141,6 +141,9 @@
       resources :burial_claims, only: %i[create show]
     end
 
+    post 'form0969', to: 'income_and_assets_claims#create'
+    get 'form0969', to: 'income_and_assets_claims#show'
+
```

Comment:
can wrap in flipper enabled check to be sure nothing can hit the endpoint
```
    if Flipper.enabled?(:income_and_assets_form_enabled)
      post 'form0969', to: 'income_and_assets_claims#create'
      get 'form0969', to: 'income_and_assets_claims#show'
    end
```

Reply:
Reversed the PR making this change, because the Flipper addition to routes.rb caused CI issues